ext/afform/core/afform.php

```diff
@@ -239,7 +239,7 @@ function afform_civicrm_pageRun(&$page) {
       'module' => $afform['module_name'],
       'directive' => _afform_angular_module_name($afform['name'], 'dash'),
     ];
-    $content = CRM_Core_Smarty::singleton()->fetchWith('afform/contactSummary/AfformBlock.tpl', ['contactId' => $cid, 'block' => $block]);
+    $content = CRM_Core_Smarty::singleton()->fetchWith('afform/InlineAfform.tpl', ['afformOptions' => ['entity_id' => $cid], 'afform' => $block]);
```

Comment:
@ufundo what about the backward compatibility of existing afforms placed as blocks?

Reply:
Yep that might be an issue. I actually thought maybe better to separate out the change to blocks, so have removed in the latest iteration.